eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -501,39 +509,68 @@ toRatifyStatePairs cg@(RatifyState _ _ _ _) =
 
 -- =============================================
 data ConwayGovState era = ConwayGovState
-  { cgProposals :: !(Proposals era)
-  , cgEnactState :: !(EnactState era)
-  , cgDRepPulsingState :: !(DRepPulsingState era)
-  -- ^ The 'cgDRepPulsingState' field is a pulser that incrementally computes the stake distribution of the DReps
+  { cgsProposals :: !(Proposals era)
+  , cgsDRepPulsingState :: !(DRepPulsingState era)
+  -- ^ The 'cgsDRepPulsingState' field is a pulser that incrementally computes the stake distribution of the DReps
   --   over the Epoch following the close of voting at end of the previous Epoch. It assembles this with some of
   --   its other internal components into a (RatifyEnv era) when it completes, and then calls the RATIFY rule
   --   and eventually returns the updated RatifyState. The pulser is created at the Epoch boundary, but does
   --   no work until it is pulsed in the 'NEWEPOCH' rule, whenever the system is NOT at the epoch boundary.
+  , cgsCommittee :: !(StrictMaybe (Committee era))
+  , cgsConstitution :: !(Constitution era)
+  , cgsCurPParams :: !(PParams era)
+  , cgsPrevPParams :: !(PParams era)
```

Comment:
Could you please put these four fields before the `cgsDRepPulsingState`? Make sure to adjust the `EncCBOR` instance too.

This will help later with implementing sharing of committee credentials when deserializing.

I know it is a bit pedantic, but, it is a very unique field, I'd like to keep it separated at least a little bit from others, so keeping it last will help with that.

I can make this change at some point later if you'd rather not deal with it now.